### ISSUE TITLE
WebView not applying CSS from link tag on iOS

### ISSUE THREAD

**Issue by XVR147** [react-native-community/react-native-webview#849]
**Bug description:**

WebView not applying external CSS file from link tag

**To Reproduce:**

Repository showing bug: https://github.com/XVR147/WebViewBug
Steps to reproduce:
1. react-native init ProjectName
2. npm i react-native-webview
3. cd ios; pod install
4. Setup WebView and related docs in React Native project
5. cd ..; react-native run-ios

**Expected behavior:**

WebView applying CSS file like on Android

**Screenshots/Videos:**

**Environment:**
 - OS: macOS 
 - OS version: 10.14.6
 - react-native version: 0.60.5
 - react-native-webview version: 7.0.5

**Notes:**

I tried adding 'css' to the list of valid extensions in metro.config.js but nothing changed besides Metro Bundler not showing warning.
Also note that CSS, via style tag or inline, in HTML file works.

**Comment by soroushm** [comment]
@XVR147 You're not able to require HTML and pass into webView
look at this example
https://aboutreact.com/load-local-html-file-url-using-react-native-webview/

**Comment by ofirgeller** [comment]
Hitting the same issue. Loading a local html file via the packager that then references a local css file inside a webview. once the packager is set up correctly the error about file not found goes away, no new error appears, but the css has no effect on the page.

I have checked the RN log, the native output with xcode, listening to error events from the webview, but getting nothing.

I tried disabling all the content origin via the webview props, no effect.
